test/integration/lib/mock-ase.ts

```diff
@@ -4,7 +4,7 @@ import {
   createAuthenticatedClient
 } from '@interledger/open-payments'
 import { AccountProvider, setupFromSeed } from 'mock-account-service-lib'
-import { createApolloClient } from './apolloClient'
+import { createApolloClient } from './apollo-client'
```

Comment:
Changed a few file names to use consistent casing (from camelCase to kebab-case). This leaves the new `integration` and `mock-account-service-lib` packages consistent at least. 

Guess it's a seperate discussion but we do a mix all over the place. I think we have more kebab-case and wouldn't mind standardizing on that going forward, whether we go back and change existing or not.

Reply:
I'm getting a type error on L45, "...Type 'ApolloLink' is not assignable to type 'ApolloLink | RequestHandler | undefined'." did you encounter this?

Reply:
I'm seeing it now after a `pnpm i`. Looks like it didnt get bumped (https://github.com/interledger/rafiki/commit/2eb0837392ac87ca997f504f904837b0b4433e7f) with the rest of the packages and there is a type mismatch between the test client version of 3.9.7 the lib (and everywhere else) of 3.9.9. 

I fixed that but I think renovate bot should have picked this up (`mock-account-service-lib` was in main, so tests were too and should have been bumped). I wonder if it's because of the renovate bot config, but I don't see what that would be looking at the `renovate.json`. Especially since nothing was necessary for `mock-account-service-lib`. It is a pnpm package. I wonder if renovate is configured not to look in the test directory? Although I dont see anything like that [in the schema](https://docs.renovatebot.com/renovate-schema.json). Cant find anything on the `config:base` which it apparently extends. https://docs.renovatebot.com/presets-config/

Reply:
yeah, guess its the test dir: https://github.com/renovatebot/renovate/discussions/12755

Reply:
I updated our `ignorePaths` in the renovate config: https://github.com/interledger/rafiki/pull/2595/commits/2f3d6cdcc6867aae908b9a56bddc39cf0fd36130

I don't really think there is a way for me to test but I believe that will work. I wish there was a way to effectively just remove a single item from the preset ignore paths but I dont see a way. I'm not sure if the `ignorePresets: [":ignorePaths"]` is required (maybe the new definition overrides it?) but without a way to test I figured I'd include it to be thorough.

Reply:
looks like it happened again 😆  

https://github.com/interledger/rafiki/commit/0b526f2147d24657dbdd3182929398df03f5c231

Reply:
> I wish there was a way to effectively just remove a single item from the preset ignore paths

Would we be able to keep the configuration the same, and extend the `ignorePaths` to use "!" as such: `!test/**`?

Reply:
Maybe... I see someone doing this here https://github.com/equinix/pulumi-equinix/blob/c0e20f3f33c7315032a2f8c459bd354cff050297/renovate.json#L6 but thats the only usage I can find.

Unfortunately I don't see a way to test this... Other than merging into main and seeing what renovate bot picks up when there is a new apollo version.

Reply:
sure, lets try that.